### PR TITLE
Feature/multigroup part8

### PR DESCRIPTION
**Pull request by whaeck**
This adds the information record and a to_file and from_file functionality

### CODE REVIEW

File: python/src/definitions.hpp [hunk 1]
```diff
@@ -142,8 +143,20 @@ void addStandardTableDefinitions( PythonClass& table ) {
...
     "    filename    the file name and path"
+  )
+  .def(
+
+    "to_file",
+    [] ( const Table& self, const std::string& filename ) {
+
+      return njoy::NDItk::toFile( self, filename );
+    },
+    "Read an ACE table from a file\n\n"
```
Comment (tgsaller) [inline python/src/definitions.hpp:154]: NDI

File: src/NDItk/base/InformationRecord.hpp [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+  /**
+   *  @brief Read the record data
+   *
+   *  @param[in] iter   the current position in the input
+   */
+  template< typename Iterator >
+  void read( Iterator& iter, const Iterator& end ) {
+
+    // move over the first endline
```
Comment (tgsaller) [inline src/NDItk/base/InformationRecord.hpp:80]: Would this endline be at the end of the info keyword?

Comment (whaeck) [reply src/NDItk/base/InformationRecord.hpp:80]: In most cases, yes. I do it this way to ensure that there are no additional non-white space characters between the info keyword and the end line (i.e. "info     stuff   \n more information" should fail but "info    \n  more information" should not)

File: python/src/definitions.hpp (hunk 1 above)
Comment (whaeck) [reply python/src/definitions.hpp:154]: copy/paste is the bane of my existence

Comment (whaeck) [reply python/src/definitions.hpp:154]: this has been addressed already (it no longer appears in the latest version). Don't know where though.

**Review by tgsaller — APPROVED** [review]

**Pull request merged**